cosmwasm/packages/wasmi-runtime/src/cosmwasm/coins.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+use crate::cosmwasm::math::Uint128;
+
+#[derive(Serialize, Deserialize, Clone, Default, Debug, PartialEq)]
+pub struct Coin {
+    pub denom: String,
+    pub amount: Uint128,
+}
```

Comment:
For this file, and other files that we copied and modified from `cosmwasm-std`, please add a comment at the top of the file (with `//! lorem ipsum` syntax) mentioning the filename of the source, and a warning/request to keep the definitions in sync.